### PR TITLE
Issue/website 89

### PR DESCRIPTION
**Pull request by scherler**
There is a small problem with the label filter, seems like css

### CODE REVIEW

**Comment by scherler** [comment]
@reviewbybees

File: Dockerfile [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
-COPY ./server/ /plugins/server/
-COPY ./node_modules/ /plugins/node_modules/
+COPY ./css/ /plugins/css/
 COPY ./webpack.commons.js /plugins/
 COPY ./webpack.config.docker.js /plugins/
-COPY ./index.html /plugins/
+COPY ./server/ /plugins/server/
 COPY ./app/ /plugins/app/
-COPY ./css/ /plugins/css/
+RUN cd /plugins/ && npm install
```
Comment (michaelneale) [inline Dockerfile]: could put WORKDIR before RUN and then wouldn't need cd *nitpick*

**Comment by michaelneale** [comment]
:bee:

File: Dockerfile (hunk 1 above)
Comment (scherler) [inline Dockerfile]: good catch I am pushing that now.

**Comment by gusreiber** [comment]
:bee:

**Pull request merged**